### DISCUSSION TITLE
Image attached in embed?

### DISCUSSION THREAD

**Discussion by BotNC** [discordjs/discord.js#5879] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I was recently trying to attach an image buffer to an embed, although `attachFiles` seems to not exist anymore.

**Describe the ideal solution**
I would want a function to attach buffers to a MessageEmbed.

**Describe alternatives you've considered**
Nothing, all the other Reddit/StackOverflow answers are using v12/v11.

**Comment by kyranet** [comment, marked as answer]
You attach buffers by adding them to the MessageOptions in this manner:

```javascript
const file = new MessageAttachment(buffer, 'image.png');
const embed = new MessageEmbed().setImage('attachment://image.png');

await channel.send({ embeds: [embed], files: [file] });
```

`MessageEmbed#attachFiles` was just an alias for the code above.
